src/util/utils.ts

```diff
@@ -1,10 +1,4 @@
-let UniformResourceLocator: any
-
-if (typeof URL === 'undefined') {
-  UniformResourceLocator = require('url').URL
-} else {
-  UniformResourceLocator = URL
-}
+const URL: any = require('url')
```

Comment:
I think the idea behind this was to keep the bundle size small by not importing the URL module via webpack

Reply:
I understand but react native doesn't have implementation for URL:
https://github.com/facebook/react-native/blob/e6057095adfdc77ccbbff1c97b1e86b06dae340b/Libraries/Blob/URL.js#L157-L159
It makes it impossible to use this library after this update

Reply:
Does react native not have the window.URL same as browsers?

Reply:
Unfortunately no